src/test/java/core/basesyntax/services/impl/CsvFileWriterImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.services.CsvFileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileWriterImplTest {
+    private static CsvFileWriter csvFileWriter;
+
+    @BeforeClass
+    public static void globalSetUp() {
+        csvFileWriter = new CsvFileWriterImpl();
+    }
+
+    @Test
+    public void createReportFileMethodTest_Ok() throws IOException {
+        List<String> expectedList = List.of("fruit,quantity",
+                "How are u doing",
+                "wass up","100%N....");
+        List<String> data = List.of("How are u doing",
+                "wass up",
+                "100%N....");
+        List<String> actualList = Files.readAllLines(Path.of("src/test/testSource/reportTest.csv"));
+        csvFileWriter.createReportFile("src/test/testSource/reportTest.csv", data);
```

Comment:
looks like we firstly read data and write after that
so what are we testing here and why did test pass?

Reply:
comment wasn't applied